quality/postman/sor-1-_table-table.postman_environment.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
please align variable naming convention in the file: now it contains mix of hyphens, underscores, camel case.